src/app/app.component.ts

```diff
@@ -17,7 +16,7 @@ export class AppComponent {
     this.currentSearchData$ = this.store.select('CurrentSearchReducer');
```

Comment:
this.currentSearchData$ is an Observable, subscription is done via async pipe in the template. Anyway as I've said before when talking about app.component.html, app.component is at this moment all with temporary code, just for displaying in the view data that at this moment has no view to display.